HIP/hip-904-frictionless-airdrop.md

```diff
@@ -8,232 +8,422 @@ type: Standards Track
+neither of those kinds of tokens.
+
+Accounts have a property, `max_auto_associations` that keeps track of the maximum number of
+automatic associations allowed by the account. Internally, the node software keeps track of
+automatic and manual associations separately, so that manually associating does not reduce the
+number of available auto association *slots*. Today, the user pays for each `max_auto_associations`
+slot, even if there is not an association. The value for `max_auto_associations` is limited to
+non-negative numbers. `0` means no automatic associations are allowed. If `max_auto_associations` is
+a positive number, then only that many automatic associations are permitted. A separate
+property, `used_auto_associations`, keeps track of how many automatic associations were used.
+If `max_auto_associations` is some number, and then later is switched to a number N, such that N
+is < `used_auto_associations`, then no new automatic associations are possible. Manual associations
+are always possible, regardless of the values of `max_auto_associations`
+and  `used_auto_associations`. This current logic is confusing and not fully transparent.
 
 This HIP changes this behavior in the following ways:
 
 - The value `-1` is now permitted for `max_auto_associations`, and this value means “unlimited”.
-- The account holder no longer prepays for unused slots regardless of the setting for `max_auto_associations`. Slots are only paid for when used, and are initially paid for by the sender during automatic associations. The account is responsible for covering rent for each association after the first auto-renewal period. `used_auto_associations` is incremented every time an auto-association is made.
-- The default value for used_auto_associations for new automatically-created accounts will now be -1. This means, if an account is created automatically (by performing a token transfer to an alias that does not yet exist, see [HIP-583](https://hips.hedera.com/hip/hip-583)), then the automatically created account is configured with unlimited automatic token associations. Accounts created by using the [AccountCreateTransaction](https://docs.hedera.com/hedera/sdks-and-apis/sdks/accounts-and-hbar/create-an-account) (HAPI: [CryptoCreate](https://github.com/hashgraph/hedera-protobufs/blob/main/services/crypto_create.proto))
-will continue to have a used_auto_associations default of 0. Accounts created before this feature is released will remain unchanged.
+- The account holder no longer prepays for unused slots regardless of the setting
+  for `max_auto_associations`. Slots are only paid for when used, and are initially paid for by the
+  sender during automatic associations. The account is responsible for covering rent for each
+  association after the first auto-renewal period. `used_auto_associations` is incremented every
+  time an auto-association is made.
+- The default value for used_auto_associations for new automatically-created accounts will now be
```

Comment:
Shouldn't it be that the default value of `max_auto_associations` for new automatically created accounts will be -1, not the `used_auto_associations`.